extra/generate-display-names/display-names

```diff
@@ -185,7 +185,8 @@
 * ../../tables/ta.tbl                               Tamil, computer                                    Tamil computer braille
 * ../../tables/te.tbl                               Telugu                                             Telugu braille
 * ../../tables/th-comp8-backward.utb                Thai, computer                                     Thai computer braille
-* ../../tables/th-g0.utb                            Thai                                               Thai braille
+  ../../tables/th-g0.utb                            Thai, uncontracted                                 Thai braille grade 0
+  ../../tables/th-g1.utb                            Thai, partially contracted                         Thai braille grade 1
```

Comment:
@egli "Thai braille grade 0" and "Thai braille grade 1" is not consistent with how tables are generally named. Change the first column to a `*` to see how the table name generator tool would do it.

Reply:
I think I fixed it on master